luigi/contrib/kubernetes.py

```diff
@@ -62,6 +62,9 @@ class kubernetes(luigi.Config):
     kubernetes_namespace = luigi.OptionalParameter(
         default=None,
         description="K8s namespace in which the job will run")
+    max_retrials_to_get_pods = luigi.IntParameter(
```

Comment:
`retrials` isn't the word you intend. I think you mean `retries`

and to that end, can the var name just be `max_retries`?